README.adoc

```diff
@@ -18,7 +18,7 @@ ifdef::env-github[]
 endif::[]
 // Project URLs:
 :url-project: https://github.com/riptano/docs-ui
-:url-preview: https://riptano.github.io/docs-ui/
+:url-preview: https://fantastic-adventure-593lpwy.pages.github.io/main/home/index.html
```

Comment:
I think we should change this to https://riptano.github.io/datastax-docs-site/main
two reasons 1) /home is the old build - need to have GH pages clean that up
2) the URL https://riptano.github.io/datastax-docs-site/main will auto redirect to fantastic-adventure, and if that should change for any reason, we don't need to worry about the README being out of date

Reply:
Great catch!